docs/content/users/usage/cli.md

```diff
@@ -62,7 +62,8 @@ To import static file assets for a project, such as uploaded images and document
 * For Drupal projects, these are the `sites/default/files` and `../private` directories.
 * For Magento 1 projects, this is the `media` directory.
 * For Magento 2 projects, this is the `pub/media` directory.
-* For Shopware projects, this is the `media` directory.
+* For Shopware 5 projects, this is the `media` directory.
```

Comment:
I'm not sure exactly how this works since there isn't explicit support for Shopware 5. Maybe:

```suggestion
* For Shopware 5 projects (usually of type `php`), this will be the `media` directory.
```

Reply:
I guess the problem here is that unless you set `upload_dirs` then DDEV won't do anything special with Shopware 5.